### PR TITLE
Set up modules

### PR DESCRIPTION
**Pull request by KUBAHO3**
In this Project:

- I have created different modules and linked them with the main index.js
- I have solved all linter errors in the project
- I have added all previous functionalities of awesome books
- I have made use of Es6 and followed all requirements

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <div class="main-container">
+      <header>
+        <nav class="container">
+          <p class="logo">AwesomeBook</p>
+          <ul>
+            <li class="listBtn">List</li>
+            <li class="add-new-btn">Add new</li>
+            <li class="contact">Contact</li>
+          </ul>
+        </nav>
```
Comment (joel-kalema) [inline index.html:20]: Good job working on the ES6 😃, but you need to fix some parts 👍.

- [ ]  Currently your `nav links` doesn't work and I can't see the list of books, kindly, fix it 👍 
- [ ] Also the current date doesn't display in the navigation bar, kindly fix it 👍 

books

**Review by joel-kalema — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (KUBAHO3) [reply index.html:20]: Have you installed the packages correctly? @joel-kalema

Comment (joel-kalema) [reply index.html:20]: yes i did @KUBAHO3

Comment (KUBAHO3) [reply index.html:20]: Yes! Thank you for the nice review and the chance to connect with you to hear me out!

**Comment by KUBAHO3** [comment]
Please! for the next reviewer consider installing node packages before running the project! And again make use of liveServer VsCode extension to ensure the connection between modules! Do not hesitate to contact me when needed!
Thank you!

**Review by levy002 — APPROVED** [review]
## Hello @KUBAHO3 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Highlights
- Professional PR :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**